assets/dev/js/frontend/handlers/video.js

```diff
@@ -58,7 +58,10 @@ export default class Video extends elementorModules.frontend.handlers.Base {
 			$videoIframe.attr( 'src', lazyLoad );
 		}
 
-		$videoIframe[ 0 ].src = this.apiProvider.getAutoplayURL( $videoIframe[ 0 ].src );
+		if ( this.getElementSettings( 'autoplay' ) ) {
+			$videoIframe.attr( 'allow', 'autoplay' );
```

Comment:
@JxxIT 
I tested your PR on my localhost and it wasn't always working for me. Sometimes it worked, and sometimes it didn't.

I remember that long ago I worked on this website https://www.werkenbijvlastuin.nl/, and there we fixed it like this:

<iframe width="560" height="315" src="https://www.youtube.com/embed/VY5s2O0eIZk?rel=0&amp;controls=0&amp;showinfo=0&amp;mute=1&amp;autoplay=1&amp;loop=1&amp;playlist=VY5s2O0eIZk" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen=""></iframe>

Meaning, there we added the autoplay to both the URL(`autoplay=1`) and to the `allow` attribute.

What do you think about that?

Reply:
@hein-obox
Hi, is it possible that it sometimes isn't working because of [chrome's autoplay policy](https://developer.chrome.com/blog/autoplay)? I already did autoplay=1, with `$videoIframe[ 0 ].src = 
this.apiProvider.getAutoplayURL( $videoIframe[ 0 ].src );` (line 63)

Reply:
@JxxIT Does the `this.apiProvider.getAutoplayURL( $videoIframe[ 0 ].src );` always get applied? 
I tried to add breakpoints, but they weren't triggering, and I didn't see the `autoplay=1` inside the iframe URL.

Reply:
@JxxIT I will test it again tomorrow.

Reply:
@hein-obox Okay thanks!

Reply:
@hein-obox Hi, is it now working for you?

Reply:
@hein-obox Can you test it?

Reply:
@hein-obox Hi, can you test it?

Reply:
@JxxIT I have studied it in more detail, and I realized that the autoplay only works when the video is muted, which also explains the difference with the url that I gave in the example.